components/director/internal/domain/scenarioassignment/service.go

```diff
@@ -50,14 +59,20 @@ func (s *service) Create(ctx context.Context, in model.AutomaticScenarioAssignme
 		return model.AutomaticScenarioAssignment{}, err
 	}
 	err = s.repo.Create(ctx, in)
-	switch {
-	case err == nil:
-		return in, nil
```

Comment:
on `err == nil` you can do nothing and keep `switch` statement

Reply:
Yup, but I think in this way it is simpler. the `switch` looked good when it was at the end of the method. Of course that's just a personal preference.